### PR TITLE
kdenlive: update to 22.04.1

### PR DESCRIPTION
**Pull request by hervyqa**
- add opencv (for motion tracking)
- remove vlc (only as optional media player)

#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/kdenlive/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'kdenlive'
 pkgname=kdenlive
-version=21.12.3
+version=22.04.0
```
Comment (ericonr) [inline srcpkgs/kdenlive/template:3]: .1 seems to be available.

**Comment by ericonr** [comment]
Is there a good enough replacement for VLC?

**Comment by hervyqa** [comment]
> Is there a good enough replacement for VLC built into it now?

It seems that there is no need to add a media player like vlc. I see from some other linux distributions do not require media player like vlc, such as [archlinux](https://archlinux.org/packages/extra/x86_64/kdenlive/), [solus](https://dev.getsol.us/source/kdenlive/browse/master/package.yml), and [openSUSE](https://build.opensuse.org/package/view_file/openSUSE:Factory/kdenlive/kdenlive.spec?expand=1).

File: srcpkgs/kdenlive/template (hunk 1 above)
Comment (hervyqa) [reply srcpkgs/kdenlive/template]: Done :)

**Pull request merged**